### ISSUE TITLE
Notification of records marked 'Plausible' in the wrong section of the email

### ISSUE THREAD

**Issue by DavidHepper** [BiologicalRecordsCentre/iRecord#1028]
When a verifier marks a record Plausible the recorder/reporter receives an email notification but in the example below these are reported in the 'Record Cleaner results...' rather than in the 'Verification of your records'. Is this intentional? I had to refer to the record detail to find out what had happened. In the three cases shown the record had indeed failed one automated check but that event was a month ago and was not the change of status to Plausible that triggered the email. 
I suggest reporting Plausible in the 'Verification of your records' section of the notification.
![image](https://user-images.githubusercontent.com/24915271/110447711-64e50300-80b8-11eb-8745-2470ae62e994.png)

**Comment by DavidHepper** [comment]
Further to this, the same Record IDs are _also_ listed in the 'Verification of your records' section.

**Comment by kitenetter** [comment]
@DavidHepper my understanding is that the first three examples in your screenshot are being notified because of the failure to pass a rulecheck, and the fact they are "Plausible" is incidental to that.

Can I ask if you are sure that users do get notified when a record is marked as "Plausible"? I was under the impression that they didn't but I'm not sure I have any evidence for that. I will test it with one of my own records, but if you have seen notifications of "Plausible" under the "Verification of your records" heading could you let me know please.

**Comment by DavidHepper** [comment]
You are indeed correct that the upper section of the email is reporting the Flight Period rule failure.

In answer to your extra question, these records really are reported in the **Verification of your Records** section below. Here's an example from a different email where two 'Plausible' markings are reported back to the CMS User, along with many others:

![image](https://user-images.githubusercontent.com/24915271/112365638-c9ff4200-8ccf-11eb-9109-0a906ceb6e04.png)

**Comment by kitenetter** [comment]
Thanks @DavidHepper.

It looks like verification decisions are being shown under the verification heading, and rule failures under the Record Cleaner heading, which seems to be as expected.

Is there still an issue to resolve here?

**Comment by DavidHepper** [comment]
Yes, this notification in section **Record cleaner results for your records** is only _issued now_ because the record has just been marked _Plausible_. It is likely that the same Record Cleaner failure was previously notified when the record was first entered (but I cannot prove this, due to another unrelated issue with notifications - #784). This second notification is therefore unnecessary because the marking as _Plausible_ is appropriately included in the lower section **Verification of your records** of the same email.

**Comment by kitenetter** [comment]
@johnvanbreda it appears that if a record is marked as Plausible, two things are happening:
1. A notification is sent that the verification status has been applied
2. A notification is re-sent for any record cleaner ruole checks, regardless of how long ago the record was originally entered

The first of those is correct behaviour, the second is unnecessary.

Can you check the code to see if record cleaner checks are being triggered unnecessarily?
